### PR TITLE
fix(scss): fixed wrong scss selector issues in storybook

### PR DESCRIPTION
**Pull request by RaphaelAmadeu**
### Description

I've found a scss problem related to the `ContentGroup` on the `ListSection` that was affecting another components, I've fixed this and also changed the selector made for testing purposes on the `ContentGroup` story scss file.

### Changelog

**Changed**

- SCSS code

### CODE REVIEW

File: packages/styles/scss/patterns/blocks/listsection/_listsection.scss [hunk 1]
```diff
@@ -76,6 +76,17 @@
...
+      padding-left: 0;
+    }
+
+    .#{$prefix}--content-group {
+      padding-bottom: $carbon--layout-03;
+      &:last-of-type {
+        padding-bottom: 0;
+      }
+    }
+
```
Comment (jeffchew) [inline packages/styles/scss/patterns/blocks/listsection/_listsection.scss]: Adding @kennylam here, as he's working on the content block at the moment which includes refactoring listsection.

File: packages/styles/scss/patterns/blocks/listsection/_listsection.scss [hunk 2]
```diff
@@ -76,6 +76,17 @@
       }
     }
 
+    .#{$prefix}--content-group__title {
+      padding-left: 0;
```
Comment (kennylam) [inline packages/styles/scss/patterns/blocks/listsection/_listsection.scss]: @RaphaelAmadeu I'm seeing a lot of this in the `UseCases` pattern I'm currently refactoring. Styles like this aren't really necessary when using Carbon's grid. I've been removing stuff like this wherever I see it.

File: packages/react/src/patterns/sub-patterns/ContentGroup/__stories__/index.scss [hunk 3]
```diff
@@ -1,6 +1,6 @@
 @import '../../../../../../styles/scss/patterns/sub-patterns/content-group/content-group';
```
Comment (photodow) [inline packages/react/src/patterns/sub-patterns/ContentGroup/__stories__/index.scss]: Isn't this already being imported?

File: packages/react/src/patterns/sub-patterns/ContentGroup/__stories__/ContentArrayGroup.stories.js [hunk 4]
```diff
@@ -19,5 +19,9 @@ storiesOf('Patterns (Sub-Patterns)|ContentGroup', module)
       'Lorem ipsum dolor sit amet.'
     );
 
-    return <ContentGroup heading={copy} />;
+    return (
+      <div className="content-group-story">
+        <ContentGroup heading={copy} />
+      </div>
```
Comment (photodow) [inline packages/react/src/patterns/sub-patterns/ContentGroup/__stories__/ContentArrayGroup.stories.js]: Could we leverage the grid here instead? Similar to how we're doing in `FeatureCard`?

```html
<div class="bx--grid ">
  <div class="bx--row">
    <div class="bx--col-sm-4 bx--col-lg-8 bx--offset-lg-4">
      ...
    </div>
  </div>
</div>
```

File: packages/styles/scss/patterns/blocks/listsection/_listsection.scss (hunk 2 above)
Comment (photodow) [reply packages/styles/scss/patterns/blocks/listsection/_listsection.scss]: 👆

File: packages/react/src/patterns/sub-patterns/ContentGroup/__stories__/ContentArrayGroup.stories.js [hunk 5]
```diff
@@ -19,5 +19,9 @@ storiesOf('Patterns (Sub-Patterns)|ContentGroup', module)
       'Lorem ipsum dolor sit amet.'
```
Comment (photodow) [inline packages/react/src/patterns/sub-patterns/ContentGroup/__stories__/ContentArrayGroup.stories.js]: Could we rename `ContentArrayGroup.stories.js` to `ContentGroup.stories.js`?

File: packages/styles/scss/patterns/blocks/listsection/_listsection.scss [hunk 6]
```diff
@@ -76,6 +76,17 @@
       }
     }
 
+    .#{$prefix}--content-group__title {
```
Comment (jeffchew) [inline packages/styles/scss/patterns/blocks/listsection/_listsection.scss]: Is this file getting renamed or that's being addressed in another PR?

**Pull request closed without merging**